lib/codecs.py

```diff
@@ -19,6 +19,7 @@ class Codec(str, enum.Enum):
   MJPEG = "mjpeg"
   MPEG2 = "mpeg2"
   VC1   = "vc1"
-
+  VVC_8   = "vvc_8"
+  VVC_10  = "vvc_10"
```

Comment:
This should not include the bitdepth indicator.
```suggestion
  VVC   = "vvc"
```

Reply:
You need to add `Codec.VVC` to the `capcodec` condition (i.e. "caps lookup translation")... located at the top of the decode_test_class function in lib/gstreamer/va/decoder.py

Reply:
Thank you for the review, it has been updated.